README.rst

```diff
@@ -762,12 +762,13 @@ Exit codes
 
 Exit code is build using following bit flags:
 
-*  0 OK
-*  4 Some files have invalid syntax (only in dry-run mode)
-*  8 Some files need fixing (only in dry-run mode)
-* 16 Configuration error of the application
-* 32 Configuration error of a Fixer
-* 64 Exception raised within the application
+*  0 OK.
+*  1 PHP/HHVM minimal requirement not matched.
```

Comment:
exit status `1` (and 2, and 128+) has special meaning, that's why we don't use it inside application since v2.
`Catchall for general errors`.
if error occurs in our app - we provide proper error.
if sth goes really wrong - we use `1`.

Maybe sth like `general error occur, eg pHP/HHVM ... not matched` ?